python/pynq/mmio.py

```diff
@@ -38,6 +38,7 @@
 import mmap
 import math
 from . import general_const
+import numpy as np
```

Comment:
The argument behind using numpy is that it provides a way to view an mmapped file as an array of 32-bit ints which is essentially what we have rewritten in a custom way. I don't think it is possible to do the same using the array class. I experimented with various ways of reading and writing and numpy offered by far the best total performance, any additional features are just bonuses.